src/components/Proposal/ProposalDetailsPage.tsx

```diff
@@ -97,6 +97,28 @@ class ProposalDetailsPage extends React.Component<IProps, IState> {
     this.setState({ showVotersModal: false });
   }
 
+  private parseYouTubeVideoIdFromUri = (url: string): string => {
+    const videoid = url.match(/(\/|%3D|v=)([0-9A-z-_]{11})([%#?&]|$)/);
+    if(videoid && (videoid.length >= 3)) {
```

Comment:
space after the if

Reply:
hmm, seems like our linter ought to catch that.

fixed, and note I also improved the error handling logic